### ISSUE TITLE
[Question] Question on STFT loss

### ISSUE THREAD

**Issue by OnceJune** [TensorSpeech/TensorFlowTTS#484]
https://github.com/TensorSpeech/TensorFlowTTS/blob/c0377a34ba04c9ebb962b4057ece7bc48a57559e/tensorflow_tts/losses/stft.py#L55
Hi, when read the paper https://arxiv.org/pdf/1910.11480.pdf, I found the mag loss is calculated by divide number of elements in the magnitude，but the code didn't do this.
https://github.com/kan-bayashi/ParallelWaveGAN apply this by using torch.nn.functional.l1_loss, which outputs mean by default.

https://github.com/kan-bayashi/ParallelWaveGAN/blob/53d14969089b3d3229fe6bfce221234f25a9d836/parallel_wavegan/losses/stft_loss.py#L82

**Comment by dathudeptrai** [comment]
@OnceJune it's not important, the loss will be reduce_mean by default in Trainer.

**Comment by OnceJune** [comment]
@dathudeptrai thanks
